### PR TITLE
App.yaml in kubernetes step has wrong indentation for template

### LINKED ISSUE #1673: app.yaml wrong indentation
app.yaml wrong indentation causing error when `kubectl apply -f app.yaml`

## Expected Behavior

```
kubectl apply -f app.yaml                                                                 [13:49:02]
deployment.apps/backstage created
```

## Current Behavior

```
$kubectl apply -f app.yaml                                                                 [13:48:57]

error: error validating "app.yaml": error validating data: [ValidationError(Deployment.spec.selector): unknown field "template" in io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector, ValidationError(Deployment.spec): missing required field "template" in io.k8s.api.apps.v1.DeploymentSpec]; if you choose to ignore these errors, turn validation off with --validate=false
```
## Possible Solution
see fix: 

## Steps to Reproduce
```
cd install/kubernetes
kubectl apply -f app.yaml     
```
## Context

trying to launch kuberenetes deployment of backstage for testing

## Your Environment

- NodeJS Version (v12)
- kubectl version (v1.18.2)
- Operating System and Version (MacOS Catalina)

### PR DESCRIPTION
**Pull request by timurista**
## App.yaml wrong indentation for template
app.yaml in kubernetes step has wrong indentation for template
https://github.com/spotify/backstage/blob/e888e1865d81a1e147444c85a167989b1a332abe/install/kubernetes/app.yaml#L28

was getting this:
```
error: error validating "app.yaml": error validating data: [ValidationError(Deployment.spec.selector): unknown field "template" in io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector, ValidationError(Deployment.spec): missing required field "template" in io.k8s.api.apps.v1.DeploymentSpec]; if you choose to ignore these errors, turn validation off with --validate=false
```
Problem was indentation
with the fix:
```
$ kubectl apply -f app.yaml                                                                 [13:49:02]
deployment.apps/backstage created
```

Fixes #1673

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by timurista** [comment]
@ daviddyball maybe you can review?

File: install/kubernetes/app.yaml [hunk 1]
```diff
@@ -11,17 +11,17 @@ spec:
...
+  template:
+    metadata:
+      labels:
+        app: backstage
+        component: frontend
+    spec:
+      containers:
+      - name: app
+        image: spotify/backstage:latest
+        imagePullPolicy: IfNotPresent
```
Comment (timurista) [inline install/kubernetes/app.yaml:23]: allows you to pull local image. But I don't think spotify publishes a public image for backstage. All I see is other backstage images from various users which appear out of date.

File: install/kubernetes/backend.yaml [hunk 2]
```diff
@@ -11,17 +11,17 @@ spec:
...
+  template:
+    metadata:
+      labels:
+        app: backstage
+        component: backend
+    spec:
+      containers:
+      - name: backend
+        image: spotify/backstage-backend:latest
+        imagePullPolicy: IfNotPresent
```
Comment (timurista) [inline install/kubernetes/backend.yaml:23]: same policy to allow users to build and use local file with something like docker-desktop kubernetes cluster

**Review by Rugvip — APPROVED** [review]
Makes sense, thanks! 👍 

Please remove the adopters reformatting change though 😅

**Pull request merged**